enos/modules/test_e2e_ui/main.tf

```diff
@@ -63,6 +63,16 @@ variable "target_port" {
   type        = string
   default     = ""
 }
+variable "target_ca_key" {
```

Comment:
Maybe call it `target_ca_key_private` for the sake of being as explicit as with the `target_ca_key_public`?